web/Dockerfile

```diff
@@ -32,6 +32,9 @@ RUN pnpm install --frozen-lockfile
 ENV DOCKER_BUILD 1
 ENV NEXT_MANUAL_SIG_HANDLE true
 
+# set commit hash environment variable
+RUN echo "NEXT_PUBLIC_BUILD_ID=$(git rev-parse HEAD)" >> /etc/environment
```

Comment:
This will fail if git is not available for example. Looking at the Porter deployments, i think we can use this here: https://github.com/langfuse/langfuse/blob/2e4eb409e9cc28b02531760db9bbff113cae0e7e/.github/workflows/porter_stack_web_us_temp.yml#L28-L29

Reply:
If it does not exist, i would fall back to the uuid in generateBuildId

Reply:
Acc to porter docs we can add this here as `PORTER_NEXT_PUBLIC_BUILD_ID` for langfuse cloud deployments

> If it does not exist, i would fall back to the uuid in generateBuildId

this does not seem to be correct as the trpc logic exclusively uses the env and not the next build id